### ISSUE TITLE
AppName.exe.config breaking incremental build as it is not modified when the application builds

### ISSUE THREAD

**Issue by xEsteem** [dotnet/project-system#7931]
I have 2 projects, ProjectA, a net472 console application, and ProjectB, a net472 class library.
I build project A from scratch at say 8:50am, it also creates a ProjectA.exe.config file that also gets placed in the build/release/net472 folder. I do not create this config file at all, and its not in the project. It seems as though it is being built automatically during the build, and doesnt contain much

```
<?xml version="1.0" encoding="utf-8"?>
<configuration>
  <startup>
    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.7.2" />
  </startup>
</configuration>
```

If i make a change to ProjectB and build it at 8:55am, and then at say 9:00,I build ProjectA, it will build as one of its dependencies has changed. All of the new binaries are copied into ProjectA's build/release/net472 folder **HOWEVER** the ProjectA.exe.config timestamp remains the same as when it was created initially during the first build at 8:50am.

If i build ProjectA again, the FastUpToDate check is failing because the ProjectA.exe.config's last modified timestamp (8:50) is older than ProjectB binaries which were built at 8:55am, even though all ProjectA binaries are up to date from the last build.
I can build ProjectA hundreds of times, even "Rebuild", and nothing will update the ProjectA.exe.config file, so it will never pass the FastUpToDate and will always build, and therefore incremental build is broken.

```
4>FastUpToDate: Input '...ProjectB.dll' is newer (10/02/2022 08:55:00) than earliest output '...ProjectA.exe.config' (10/02/2022 08:50:00), not up to date. (ProjectA)
```

The only ways i can think of getting around this are adding a post build target to ProjectA that literally just resaves the ProjectA.exe.config in the /bin/release/net472 folder to update the timestamp when it builds so that it does not keep triggering the build. But that seems like a hacky step.

I am building using 
```
Microsoft Visual Studio Professional 2022 (64-bit) 
Version 17.0.5
```
however the issue is also present on VS2019

**Comment by drewnoakes** [comment]
@xEsteem I believe this is a duplicate of https://github.com/dotnet/project-system/issues/6758.

The good news is that bug was fixed in 17.1 (via #7810).

Are you able to verify that the issue doesn't repro for you in 17.1?

**Comment by drewnoakes** [comment]
Closing as fixed. Please report back if you still see this in 17.1 or later. Thanks.
